bitnami/jaeger/values.yaml

```diff
@@ -0,0 +1,882 @@
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
+
+## Secret with keystore, keystore password, truststore, truststore password
+## DEPRECATED. Use tls.existingSecret instead
+# tlsEncryptionSecretName:
+
+## Database credentials
+## @param dbUser.user Jaeger admin user
+## @param dbUser.forcePassword Force the user to provide a non
+## @param dbUser.password Password for `dbUser.user`. Randomly generated if empty
+## @param dbUser.existingSecret Use an existing secret object for `dbUser.user` password (will ignore `dbUser.password`)
+##
+dbUser:
+  user: jaeger
+  forcePassword: false
+  password: ""
+  ## Use an existing secrets which already stores your password data.
+  ## for backwards compatibility, existingSecret can be a simple string,
+  ## referencing the secret by name.
+  ## existingSecret:
+  ##   ## Name of the existing secret
+  ##   ##
+  ##   name: mySecret
+  ##   ## Key mapping where <key> is the value which the deployment is expecting and
+  ##   ## <value> is the name of the key in the existing secret.
+  ##   ##
+  ##   keyMapping:
+  ##     jaeger-password: myJaegerPasswordKey
+  ##
+  existingSecret: ""
+
+## @param initDBConfigMap ConfigMap with cql scripts. Useful for creating a keyspace and pre-populating data
+##
+initDBConfigMap: ""
+## @param initDBSecret Secret with cql script (with sensitive data). Useful for creating a keyspace and pre-populating data
+##
+initDBSecret: ""
+## @param existingConfiguration ConfigMap with custom jaeger configuration files. This overrides any other Jaeger configuration set in the chart
+##
+existingConfiguration: ""
+## Cluster parameters
+## @param cluster.name Jaeger cluster name
+## @param cluster.seedCount Number of seed nodes
+## @param cluster.numTokens Number of tokens for each node
+## @param cluster.datacenter Datacenter name
+## @param cluster.rack Rack name
+## @param cluster.endpointSnitch Endpoint Snitch
+## @param cluster.internodeEncryption DEPRECATED: use tls.internode and tls.client instead. Encryption values.
+## @param cluster.clientEncryption Client Encryption
+## @param cluster.extraSeeds For an external/second jaeger ring.
+## @param cluster.enableUDF Enable User defined functions
+##
+cluster:
+  name: jaeger
+  seedCount: 1
+  numTokens: 256
+  datacenter: dc1
+  rack: rack1
+  endpointSnitch: SimpleSnitch
+  ## NOTE: internodeEncryption requires tlsEncryptionSecretName
+  ##
+  internodeEncryption: none
+  clientEncryption: false
+  ## eg:
+  ## extraSeeds:
+  ##   - hostname/IP
+  ##   - hostname/IP
+  ##
+  extraSeeds: []
+  enableUDF: false
+
+
+
+
+
+
+
+
+
+query:
+  ## @param command Command for running the container (set to default if not set). Use array form
+  ##
+  command: ["--cassandra.keyspace=jaeger_v1_dc1", "--cassandra.servers=cassandra"]
```

Comment:
This is used for overriding the default command with another one (even though the name can be misleading, but it is a standard now and difficult to change). Leave this one empty and add it to the template YAML

Reply:
Done [3f38b9e](https://github.com/bitnami/charts/pull/13480/commits/3f38b9e83a9e62e131c925fc67ab470d850178e7)